packages/phones/src/apn/APN.ts

```diff
@@ -14,13 +13,12 @@ export function loadAPN(): Promise<APNStatic> {
     'awesome-phonenumber'
   ).then(
     (apn) => {
-      loadedAPN = apn.default;
-
+      loadedAPN = apn.parsePhoneNumber;
       return loadedAPN;
     },
-    (error: Error) => {
+    (error) => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
is there a way to avoid disabling the rule?

Reply:
@tokarevfedor1991 I could not found any solution to avoid disabling. Do you have any suggestion?

Reply:
I think you can leave `error: Error` as before